src/Servers/HttpSys/src/RequestProcessing/Request.cs

```diff
@@ -115,8 +115,8 @@ internal Request(RequestContext requestContext)
     internal ulong RawConnectionId { get; }
 
     // No ulongs in public APIs...
-    public long ConnectionId => (long)RawConnectionId;
-
+    public long ConnectionId => RawConnectionId != 0 ? (long)RawConnectionId : (long)ConnectionId;
```

Comment:
Should this be `UConnectionId` instead?

```suggestion
    public long ConnectionId => RawConnectionId != 0 ? (long)RawConnectionId : (long)UConnectionId;
```

Reply:
thanks.